build/Dockerfile

```diff
@@ -1,8 +1,20 @@
+FROM openshift/origin-release:golang-1.10
+
+RUN mkdir -p /go/src/github.com/embercsi/ember-csi-operator/
+RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+COPY . /go/src/github.com/embercsi/ember-csi-operator/
+WORKDIR /go/src/github.com/embercsi/ember-csi-operator/
+
+RUN make dep
+RUN make compile
+
 FROM centos:7
 
+RUN yum update -y; yum clean all
```

Comment:
We should use `&&` instead of `;` so we fail if the update fails.

Reply:
yep, good point!